src/v/kafka/server/replicated_partition.cc

```diff
@@ -633,11 +633,9 @@ ss::future<error_code> replicated_partition::validate_fetch_offset(
 
         const std::pair<model::offset, model::offset> bounds = std::minmax(
```

Comment:
nit: could be just min then if we don't need the max

Reply:
👍🏼